pubspec.yaml

```diff
@@ -13,13 +13,17 @@ dependencies:
       url: git@github.com:linagora/linkfy_text.git
       ref: main
 
+  flutter_math_fork:
```

Comment:
Add todo here to link the issue, and will back to upstream whenever we upgrade to 3.29